components/auth/auth-wrapper.tsx

```
@@ -0,0 +1,74 @@
+  }
+
+  // If not signed in, show a message
+  if (!isSignedIn) {
+    return (
+      <div className="container py-8">
+        <div className="bg-amber-50 border border-amber-200 rounded-md p-4 text-amber-800">
+          <h3 className="font-medium text-lg mb-2">Authentication Required</h3>
+          <p className="mb-4">
+            You need to be signed in to access this page.
+          </p>
+          <a href="/sign-in" className="px-4 py-2 bg-amber-200 text-amber-800 rounded hover:bg-amber-300 transition-colors">
```

ESLint

[error] Do not use an `<a>` element to navigate to `/sign-in/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)